modules/backend/classes/FormField.php

```diff
@@ -243,6 +247,24 @@ public function options($value = null)
         return $this;
     }
 
+    /**
+     * Returns the label(s) associated with a field value(s) for the field options
+     */
+    public function valueLabel(): string|array
+    {
+        $defaultLabel = '';
+
+        if (!($options = $this->options()) || is_null($this->value)) {
```

Comment:
@LukeTowers side note - there's a PHPCS rule that prevents side effects within a condition definition (such as defining a variable in an `if` statement).

I'd be keen to use it because I really am not a fan of this :stuck_out_tongue: 

```suggestion
        if (!($options = $this->options()) || is_null($this->value)) {
```

Reply:
I'm not a fan of this PHPCS rule... 😛